docs/user-guide/framework/bootstrap-components/templating.md

```diff
@@ -0,0 +1,56 @@
+# Templating
+
+Templating is a feature offered to drivers to inject property values into infrastructure and lifecycle scripts. For example, the execution of a lifecycle script may vary based on a request property called `fileName`, so uses a variable placeholder:
+
+```
+touch {{ fileName }}
+```
+
+Rendering the template produces a different result based on the property context used:
+
+Property Context:
+```
+{
+    'fileName': 'my-file.txt'
+}
+```
+
+Script:
+```
+touch my-file.txt
+```
+
+The [Jinja](https://jinja.palletsprojects.com/en/2.11.x/templates/) library is used to process templates.
+
+Although optional, the Services listed below are enabled by default when using the `ignition.boot.api.build_vim_driver` or `ignition.boot.api.build_lifecycle_driver` method. 
+
+## Services
+
+The following services are auto-configured when enabled:
+
+| Name                     | Capability         | Required Capabilities             | Bootstrap Enable/Disable flag       | Description                                                                                                                                     |
+| ------------------------ | ------------------ | --------------------------------- | ----------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| Jinja2TemplatingService | JobQueTemplatingCapabilityueCapability | - | bootstrap.templating.service_enabled | Handles rendering templates |
```

Comment:
JobQueTemplatingCapabilityueCapability - typo?

Reply:
copy and paste error :man_facepalming: